backends/aerospike.go

```diff
@@ -45,12 +45,23 @@ type AerospikeBackend struct {
 }
 
 func NewAerospikeBackend(cfg config.Aerospike, metrics *metrics.Metrics) *AerospikeBackend {
-	client, err := as.NewClient(cfg.Host, cfg.Port)
+	var hosts []*as.Host
+
+	clientPolicy := as.NewClientPolicy()
+	clientPolicy.User = cfg.User
+	clientPolicy.Password = cfg.Password
```

Comment:
Looks good to me, just one suggestion: In order to make it extra clear that `cfg.User` and `cfg.Password` are optional, can we add an if statement? Something like:
```
49
    +  var clientPolicy *as.ClientPolicy = nil
    +  if len(cfg.User) == 0 && len(cfg.Password) == 0 {
    +         clientPolicy = as.NewClientPolicy()
50  -         clientPolicy := as.NewClientPolicy()
51            clientPolicy.User = cfg.User
52            clientPolicy.Password = cfg.Password
    +  }
53
backends/aerospike.go
```

This strategy seems to be safe because `NewClientWithPolicyAndHost(policy *ClientPolicy, hosts ...*Host) (*Client, error)` initializes the policy in case it is `nil` precisely using `NewClientPolicy()`
```
80 func NewClientWithPolicyAndHost(policy *ClientPolicy, hosts ...*Host) (*Client, error) {
81     if policy == nil {
82         policy = NewClientPolicy()
83     }
~/go/pkg/mod/github.com/aerospike/aerospike-client-go@v4.0.0+incompatible/client.go
```

Reply:
It was earlier asked to remove the if statements I had a User and Password, as Golang defaults to empty.
I'll add a comment that these are optional params.

I think we should leave in the NewClientPolicy, there are a half dozen other settings that I can see getting added as we start to ramp up use of PBC around timeouts and pool management

Reply:
Sounds good. Sorry for the contracting feedback @grossjo. You're good with the code you have now.